frontend/src/pages/homepage.tsx

```diff
@@ -1,14 +1,6 @@
 import React from "react";
-import Navbar from "../utils/navbar";
 import { PageTitle } from "../utils/common";
 
 export const Homepage = () => {
-    return (
-        <>
-            <Navbar />
-            <div className={"container"}>
-                <PageTitle>Homepage</PageTitle>
-            </div>
-        </>
-    );
+    return <PageTitle>Homepage</PageTitle>;
```

Comment:
Is that all?
What is this component for?

Reply:
It's a WIP, to be changed in another PR